src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -1,5 +1,4 @@
 databaseChangeLog:
 - include:
     relativeToChangelogFile: true
-    file: db.changelog-diff-20180620-1733.yaml
-
+    file: db.changelog-diff-20180620-1733.yaml
```

Comment:
I think this needs to include the new schema changes. Except for db.changelog-diff-20180622-1820.yaml, which file was a mistake